lib/fs.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const metasync = require('metasync');
+
+const rmdirp = (dir, cb) => {
+  fs.stat(dir, (err, stats) => {
+    if (err) {
+      if (err.code === 'ENOENT') cb(null);
+      else cb(err);
+      return;
+    }
+    if (stats.isDirectory()) {
+      fs.readdir(dir, (err, files) => {
+        if (err) {
+          cb(err);
+          return;
+        }
+        files = files.map(file => path.join(dir, file));
```

Comment:
```suggestion
        const paths = files.map(file => path.join(dir, file));
```
I think it's better not to change args